### PR TITLE
Text Auth

### PR DESCRIPTION
**Pull request by andrew-brainerd**
Added ability to sign in and out from text commands `login` or `signIn` and `logout` or `signOut` respectively.

### CODE REVIEW

File: src/actions/pusher.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const PREFIX = 'PUSHER';
+
+export const CONNECTING_CLIENT = `${PREFIX}/CONNECTING_CLIENT`;
+export const UPDATING_CLIENTS = `${PREFIX}/UPDATING_CLIENTS`;
+export const SET_SYNCING = `${PREFIX}/SET_SYNCING`;
+
+export const setSyncing = isSyncing => ({ type: SET_SYNCING, isSyncing });
+
+export const connectToUpdates = channelId => async dispatch => {
+  console.log('%cConnecting to Pusher channel...', 'color: cyan');
```
Comment (ajbowen249) [inline src/actions/pusher.js:16]: Is the body of this copied in from Peapod? Might be worth stubbing it out until we actually use it.

File: src/api/game.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { basicJsonHeader, handleResponse } from './tools';
+import { parseInput } from '../utils/game';
+
+const PERKINS_API_URL = process.env.REACT_APP_PERKINS_API_URL || 'http://localhost:5000';
+
+export const createGame = async (name, createdBy = {}) => {
+  const response = await fetch(`${PERKINS_API_URL}/api/games`, {
```
Comment (ajbowen249) [inline src/api/game.js:7]: Do you have an implementation of that API in a branch in the `server` repo? I'm tempted to suggest that we set the precedent that any UI PR dependent on an API should wait for the API feature to merge as well.

**Review by ajbowen249 — DISMISSED** [review]
Couple of questions and :shipit:

File: src/api/game.js (hunk 2 above)
Comment (andrew-brainerd) [reply src/api/game.js:7]: Great point. It does have an accompanying server update to go with it, which I forgot to put a PR in for.

File: src/actions/pusher.js (hunk 1 above)
Comment (andrew-brainerd) [reply src/actions/pusher.js:16]: It is. The original purpose of this branch was to add multiplayer using this implementation, but then I realized there were many things that should come before that in order of functionality. I can remove this stuff for now.

**Pull request merged**